### PR TITLE
[4740][ADD] stock_delivery_slip_aggregate_quantity

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4740](https://www.quartile.co/web#id=4740&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: stock_delivery_slip_aggregate_quantity/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        aggregated_move_lines = super()._get_aggregated_product_quantities(**kwargs)
+        for _, line in aggregated_move_lines.items():
+            aggregated_move_quantity = self.env["aggregated.move.quantity"].create(
+                {
+                    "product_id": line["product"].id,
+                    "qty_ordered": line["qty_ordered"] or 0.0,
+                    "qty_done": line["qty_done"] or 0.0,
+                }
+            )
+            line["aggregated_move_quantity"] = aggregated_move_quantity
```
Comment (yostashiro) [inline stock_delivery_slip_aggregate_quantity/models/stock_move_line.py:20]: Update vals_list and do `create()` only once.

File: stock_delivery_slip_aggregate_quantity/reports/report_delivery_slip.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            />
+        </xpath>
+        <xpath
+            expr="//td[@name='move_line_aggregated_qty_done']/t/span[contains(@t-esc, 'qty_done')]"
+            position="attributes"
+        >
+            <attribute name="style">display: none;</attribute>
+            <span
+                t-field="aggregated_lines[line]['aggregated_move_quantity'].qty_done"
+            />
```
Comment (yostashiro) [inline stock_delivery_slip_aggregate_quantity/reports/report_delivery_slip.xml]: I suppose this is an oversight.
```suggestion
```

File: stock_delivery_slip_aggregate_quantity/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_aggregated_move_quantity,aggregated_move_quantity,stock_delivery_slip_aggregate_quantity.model_aggregated_move_quantity,base.group_user,1,1,1,1
```
Comment (yostashiro) [inline stock_delivery_slip_aggregate_quantity/security/ir.model.access.csv]: Please double-check if this should be how we define the ACL for this model, considering that the report may be downloaded by portal users.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]
Code review.

**Review by kanda999 — APPROVED** [review]
Functional review.

**Pull request merged**